Justfile

```diff
@@ -0,0 +1,13 @@
+# Build neovim
+nvim: deps
+        cmake -B build -G Ninja
```

Comment:
Note that `ninja` (sounds like "N-Jim"!) is an optional build dependency last I looked, so probably shouldn't be hard-coded here.

(It'd be different if this were an additional "if you installed ALL THE OPTIONAL deps" convenience wrapper.)